src/loans/mango.py

```diff
@@ -157,9 +197,86 @@ def process_unprocessed_events(self):
 
                 else:
                     continue
-        self.last_slot = int(time.time())
+
+            for order in account.serum3:
+                if order.market_index == 65535:
+                    # Order Unset
+                    continue
+
+                # Get base and quote token info
+                mint = self.group_token_index_map.get(mango_group)
+                if not mint:
+                    print(f'Unable to find token index map for group {mango_group}')
+                    continue
+
+                base_mint = mint.get(order.base_token_index)
+                quote_mint = mint.get(order.quote_token_index)
+
+                if not base_mint:   
+                    print(f'Unable to find base token index map for index {order.base_token_index}')
+                    continue
+
+                if not quote_mint:   
+                    print(f'Unable to find quote token index map for index {order.quote_token_index}')
+                    continue
                 
+                try:
+                    # Fetch Serum order account info
+                    fetched_order = self.client.get_account_info(order.open_orders)
+                except SolanaRpcException: 
```

Comment:
At this point, isn't it worth a try to fetch the orders again at least once after some short sleep?

Reply:
Yep